build.gradle

```diff
@@ -948,6 +948,48 @@ project(':iceberg-snowflake') {
   }
 }
 
+project(':iceberg-tencentcloud') {
+  test {
+    useJUnitPlatform()
+  }
+
+  dependencies {
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    api project(':iceberg-api')
+    implementation project(':iceberg-core')
+    implementation project(':iceberg-common')
+
+    compileOnly (libs.cos.sdk) {
+      exclude group: 'com.google.code.gson', module: 'gson'
+    }
+    compileOnly (libs.cos.sdk.bundle) {
+      exclude group: 'com.google.code.gson', module: 'gson'
+    }
+    compileOnly libs.jaxb.api
```

Comment:
are these required? I see that they have been copy/pasted from the `':iceberg-aliyun'`  module would I'd suggest to slim this down to what is required for this project